shared/package.json

```diff
@@ -1,7 +1,3 @@
 {
```

Comment:
Does this file need to exist?

Come to think of it.... do any of the shared/.../package.json files need to exist now that we've unified testing?

Reply:
Just tried removing it, and tests + app run, so think we're good!